src/pages/sitemap.xml.js

```diff
@@ -2,50 +2,54 @@ import { PROTOCOLS_API } from '../constants/index'
 import { standardizeProtocolName } from '../utils'
 
 const baseUrl = `https://defillama.com`
-const singleUrls = ["protocols", "comparison", "about", "airdrops", "chains", "recent", "nfts", "nfts/chains"]
+const singleUrls = ['protocols', 'comparison', 'about', 'airdrops', 'chains', 'recent', 'nfts', 'nfts/chains', '3defi']
```

Comment:
this is not really a new page in our site, adding this to the sitemap would require a page live on defillama.com/3defi and atm it seems that this just adds a link to the sidebar

Reply:
removed the sitemap,

the reason being I am doing the link because I don't want to mess up the packages installed, (and my frontend skill is not the strongest)